frontend/src/components/BorrowRequestsList/BorrowRequestsList.tsx

```diff
@@ -15,7 +15,6 @@ interface BorrowRequest {
 const BorrowRequestsList: FC<BorrowRequestsListProps> = () => {
 	const [apiPath, setApiPath] = useState("/borrow/request/list")
 	const { data, error, loading } = useApiGet<BorrowRequest[]>(apiPath);
```

Comment:
Keep line or I angry
```suggestion
	const [apiPath, setApiPath] = useState("/borrow/request/list")
	const { data, error, loading } = useApiGet<BorrowRequest[]>(apiPath);
	
```